oaknational/Oak-Web-Application
# feat(favicon): add oak favicon set

## Description

- Adds a favicon to the site (generated the files with favicon.io)

## Issue(s)

Fixes #414

## How to test

1. Go to {cloud link}
3. You should see the favicon appear (like screenshot below)
4. When you "add to home screen" on mobile. it should use the favicon (like screenshot below)

**NB TODO**
- this does not implement different favicons for teacher/pupil/light dark
- light/dark favicon guidance: https://stackoverflow.com/questions/55170708/detect-if-the-browser-is-using-dark-mode-and-use-a-different-favicon
- we'd probably want a react hook which contains the favicon logic when we get round to it, but it's not MVP

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/186358230-0b339c7a-e1b3-4b50-ba3a-d16ff979f4ab.png)

**before -> after**

![image](https://user-images.githubusercontent.com/12934669/186359253-3f670853-fa6c-455f-877e-4ff259e5a15a.png)


## Checklist
